workflow-templates/php-library-static.yml

```diff
@@ -0,0 +1,15 @@
+name: Static analysis
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 0 * * 1'
+ 
+jobs:
+  static:
+    uses: dvsa/.github/.github/workflows/php-static.yml@v3.2.0
```

Comment:
Probably not for this PR specifically and more as a whole; I'm unsure of the value of locking our own actions to a minor/patch tag version instead of just `@v3` - if we follow semantic versioning, we should be good to reduce the constraint while reducing the maintenance of bumping each file, each time we're doing minor/patch releases.

Reply:
have discussed this with Howard and yes think we should start to implement this @AnnaDodson  @jerotire @aaronddvsa  any thoughts?